explainers/audio-explainer.md

```diff
@@ -1,45 +1,76 @@
+
+Many specialized players support the Daisy DTBook format, but this is still a niche format, unused in the publishing industry. In the ebooks domain, the Daisy format tends to be replaced by the EPUB 3 format, with a full support from the Daisy Consortium; the same will certainly happen if a global standard emerges for audiobooks, with proper support for core accessibility features.   
+
+### Business to consumer
+
+Different platforms offer users different ways to download audiobooks: some provide apps which totally hide the format, others offer a zip download or, if the book is huge (more than 800 Mb maybe), a series of zip downloads. Such audiobooks can also be protected with some proprietary DRM, causing interoperability issues for the end-user.
+
+Alternatively, many platforms offer a "streaming" service, or more properly speaking a progressive download mp3 service, open to authentified users. The audiobook platform displays useful metadata and allows navigating from track to track. Streaming requires being online, but it avoids downloading hundreds of megabytes of data before listening to audio content. Modern streaming is based on dynamic http streaming technologies (HLS, HDS, DASH) but the mp3 format does not allow such dynamic adaptation to the available bandwith. 
+
+Even if a full set of "ONIX for Books" metadata is sent from publishers to distributors and booksellers, because mp3 metadata features are poor, end users will not benefit from useful information when they store and catalog the audiobooks they have acquired. 
+
+As there is no notion of table of contents in the audiobook structure, navigation is therefore limited to moves from one track to another. 
 
-An audiobooks spec would also help with B2B relationships. Today publishers have to deal with multiple non-standards, preparing files and metadata differently for each audiobook distributor. This results in duplicated effort, increases the chances of errors, and makes the supply chain less efficient.
 
-It is in the interest of users, creators, and distributors that we push to create a single specification for audiobooks. 
 ## Goals
-* Create a specification for the audiobook format that is usable on both the web and in packaged contexts
-* Create a specification that supports all of the major use cases of audio
-* Create a specification that is accessible
-* Create a specification that meets the needs and requirements of publishers, user agents, and users
+
+* Create a specification that is usable in a business to business use case.
+* Create a specification that is usable in an inclusive use case.
+* Create a specification that is usable in a business to consumer download use case.
+* Create a specification that is usable in a business to consumer streaming use case.
+* Create a format that is easy to author by audiovisual studios.
+* Create a format that provides a path of evolution to modern audio formats like Opus and streaming formats like Dash.
```

Comment:
I'm not sure I understand how this change is needed for the goals, this wasn't mentioned in TAG feedback. 

Also the section on Opus is unnecessary, as we don't reference file and streaming formats in the spec at all, and I like to avoid this in the explainer as much as possible.